### ISSUE TITLE
Terminal blur effect outside of window

### ISSUE THREAD

**Issue by alissonlauffer** [martyr-deepin/deepin-terminal-gtk#230]
Distro: Arch Linux
GPU: Intel HD Graphics 4400

### Screenshot
![Screen Capture_20200605053604](https://user-images.githubusercontent.com/29029632/83855443-72ff9080-a6ee-11ea-99f0-9d79e571ed88.png)

**Comment by DiegoMagdaleno** [comment]
+1 for this issue. Also affecting me in Manjaro. 

![DeepinScreenshot_select-area_20200609164210](https://user-images.githubusercontent.com/38844659/84204135-42ae4e00-aa70-11ea-9d7d-5c963dae6154.png)

**Comment by alissonlauffer** [comment]
With the new Deepin Terminal update this issue is _partially_ solved. Now the blur will go outside of the window when resizing the window.

## When making the window bigger
![Screen Capture_20200613191040](https://user-images.githubusercontent.com/29029632/84580028-a48cf180-ada9-11ea-93c1-fcffdffd3a90.png)

## When making the window smaller (Look at the corners of the window)
![Screen Capture_20200613191102](https://user-images.githubusercontent.com/29029632/84580036-b2427700-ada9-11ea-8a62-405df7b26e20.png)

**Comment by felixonmars** [comment]
Please try again with either the new version or deepin-terminal-old 5.0.4.3.

**Comment by alissonlauffer** [comment]
> Please try again with either the new version or deepin-terminal-old 5.0.4.3.

In the new version, the blur issue in my second report (https://github.com/linuxdeepin/deepin-terminal-old/issues/230#issuecomment-643684761)  still exists.

The current blur issue in deepin-terminal-old seems to have been fixed, but now there is a small glitch, where blur is being used in window corners:
![image](https://user-images.githubusercontent.com/29029632/92163565-4bfa4d00-ee0a-11ea-85bb-4709118c088e.png)

**Comment by alissonlauffer** [comment]
I think this is the same issue that makes deepin-dock borders square in Arch/Manjaro.

**Comment by felixonmars** [comment]
I see, thanks for the feedback. I have no idea where the problem could be since it's working properly here on Plasma:

![2020-09-04-051102-7c7049](https://user-images.githubusercontent.com/1006477/92168140-2a7e7d00-ee6d-11ea-84c2-d07d3ee3f765.png)

Perhaps @xmuli or others would have some ideas.

**Comment by magynhard** [comment]
@alissonlauffer and @DiegoMagdaleno do you still have that problem? If not, we could close that issue.

Here currently using the latest Manjaro Deepin 20 using deepin-terminal-gtk I can not reproduce that problem.

**Comment by alissonlauffer** [comment]
> @alissonlauffer and @DiegoMagdaleno do you still have that problem? If not, we could close that issue.
> 
> Here currently using the latest Manjaro Deepin 20 using deepin-terminal-gtk I can not reproduce that problem.

no, it has been fixed. But the blur issues described in https://github.com/linuxdeepin/deepin-terminal-gtk/issues/230#issuecomment-643684761 (with deepin-terminal) and https://github.com/linuxdeepin/deepin-terminal-gtk/issues/230#issuecomment-686739884 (with deepin-terminal-gtk) still persists.

**Comment by magynhard** [comment]
Does https://github.com/linuxdeepin/deepin-terminal-gtk/issues/230#issuecomment-643684761 also happen on this old Terminal here? Because it is about the new terminal https://github.com/linuxdeepin/deepin-terminal not the https://github.com/linuxdeepin/deepin-terminal-gtk
